LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013-2017 simplabs GmbH/Marco Otte-Witte and contributors
+Copyright (c) 2013-2017 simplabs GmbH and contributors
```

Comment:
Could we extend the copyright notice until `2018` in the same PR or would you like to update that in a follow-up?